service/matching/taskReader.go

```diff
@@ -114,16 +142,15 @@ dispatchLoop:
 }
 
 func (tr *taskReader) getTasksPump() {
-	tr.tlMgr.startWG.Wait()
```

Comment:
oh wow, that's confusing

Reply:
is there are replacement for this WG in terms of functionality? how do we ensure that this only starts when ready?

Reply:
this wait isn't necessary, because it's the last step of start

Reply:
good observation... I didn't notice